arch/arm/rpi/CMakeLists.include

```diff
@@ -9,41 +9,37 @@ set(LD_ARGUMENT -Wl,--build-id=none -Wl,-z -Wl,max-page-size=0x1000 -nostdinc -n
 set(APPEND_LD_ARGUMENTS -Wl,-lgcc)
 set(KERNEL_IMAGE_OBJCOPY )
 
-#Custom target: make sdcard
-#create an sdcard for the raspberry pi
+# sdcard: Create an sdcard for the raspberry pi
 add_custom_target(sdcard
   COMMAND ${PROJECT_SOURCE_DIR}/arch/arm/rpi/utils/makesdcard.sh "${PROJECT_SOURCE_DIR}/arch/arm/rpi/utils/boot/"
   COMMENT "Creating the sdcard"
   WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
   )
 
-#Custom target: make sdcardq
-#create an sdcard for the raspberry pi (quiet mode)
+# sdcardq: Create an sdcard for the raspberry pi (quiet mode)
 add_custom_target(sdcardq
   COMMAND ${PROJECT_SOURCE_DIR}/arch/arm/rpi/utils/makesdcard.sh "${PROJECT_SOURCE_DIR}/arch/arm/rpi/utils/boot/" y /dev/mmcblk0
-  COMMENT "Creating the sdcard (quiet mode)"
+  COMMENT "Creating the sdcard in quiet mode"
   WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
   )
 
-#Custom target: make qemu
-#Run qemu in non debugging mode
+# qemu: Run qemu in non debugging mode
 add_custom_target(qemu
   COMMAND qemu-system-arm-rpi -kernel kernel.x -cpu arm1176 -m 512 -M raspi -no-reboot -sd SWEB-flat.vmdk -device usb-kbd -device usb-mouse -serial stdio -d guest_errors,unimp
-  COMMENT "qemu-system-arm-rpi -kernel kernel.x -cpu arm1176 -m 512 -M raspi -no-reboot -sd SWEB-flat.vmdk -device usb-kbd -device usb-mouse -serial stdio -d guest_errors,unimp"
+  COMMENT "Executing `qemu-system-arm-rpi -kernel kernel.x -cpu arm1176 -m 512 -M raspi -no-reboot -sd SWEB-flat.vmdk -device usb-kbd -device usb-mouse -serial stdio -d guest_errors,unimp`"
   WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
   COMMAND reset -I
   )
 
-#Custom target: make qemu
-#Run qemu in non debugging mode
+# qemugdb: Run qemu in non debugging mode
 add_custom_target(qemugdb
   COMMAND qemu-system-arm-rpi -s -S -kernel kernel.x -cpu arm1176 -m 512 -M raspi -no-reboot -sd SWEB-flat.vmdk -device usb-kbd -device usb-mouse -serial stdio -d guest_errors,unimp
-  COMMENT "qemu-system-arm-rpi -s -S -kernel kernel.x -cpu arm1176 -m 512 -M raspi -no-reboot -sd SWEB-flat.vmdk -device usb-kbd -device usb-mouse -serial stdio -d guest_errors,unimp"
+  COMMENT "Executing `qemu-system-arm-rpi -s -S -kernel kernel.x -cpu arm1176 -m 512 -M raspi -no-reboot -sd SWEB-flat.vmdk -device usb-kbd -device usb-mouse -serial stdio -d guest_errors,unimp`"
   WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
   COMMAND reset -I
   )
 
-#Custom target: build_qemu_rpi
+# build_qemu_rpi: TODO
```

Comment:
qemu-rpi is not merged into qemu... thus you have to build qemu-rpi yourself... this target calls a script that clones qemu-rpi from github and compiles it and installs it...